pkg/features/kube_features.go

```diff
@@ -1054,7 +1054,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	TopologyManagerPolicyBetaOptions: {Default: true, PreRelease: featuregate.Beta},
 
-	TopologyManagerPolicyOptions: {Default: false, PreRelease: featuregate.Alpha},
+	TopologyManagerPolicyOptions: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
please highlight in the commit message and/or in the PR description that we want 
- to promote "topology manager policy options exist at all" (or any better phrasing you can think of) from alpha to beta. The reason I'm highlighting this is we had some initial confusion when doing the same work for cpumanager, "*ManagerPolicyOptions" and "*ManagerPolicyOptions{Alpha,Beta}" easily creates confusion
- at the same time, promote the "PreferClosestNUMANodes" option from alpha to beta

I don't think you should issue separate PRs, but I do think that clear phrasing in the commit messages/PR description and perhaps more granular commit split will make the intent clearer.